src/features/pass-survey/lib/services/SkipService.ts

```diff
@@ -1,13 +1,17 @@
 import { AbTestResponse, Answer, PipelineItem } from '@features/pass-survey';
 
+type NoParamVoidFunction = () => void;
```

Comment:
Please move to abstract folder

Reply:
@anq83 
Sorry, are we even bothering to create a separate type for this?
`() => void` is completely understandable and even shorter than `NoParamVoidFunction`